EnglishTranslation-master/SKILL.tsv

```diff
@@ -6209,44 +6209,44 @@ SKILL_20160224_006208	{#993399}{ol}[Magic]{/}{/}All Ice-type magic cast by you a
+SKILL_20160224_006229	Move while hanging on your hawk. Prevents damage from close ranged attacks and you can damage enemies with basic attacks while hanging.
 SKILL_20160224_006230	{#DD5500}{ol}[Missile]{/}{/}{nl}Fires a cannonball that spreads to target all enemies in front of you.
 SKILL_20160224_006231	Attack: 300% + #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%
 SKILL_20160224_006232	Attack: 300% + #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}INT, SPR: -#{CaptionRatio2}#
 SKILL_20160224_006233	Attack: 400% + #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}
 SKILL_20160224_006234	Blocks: #{CaptionRatio}# times{nl}Magic Circle Duration: 20 seconds
-SKILL_20160224_006235	Plant a Oguon glyph on the ground symbolizing a strength noir. Increases STR and AoE attack ratio of your zombies nearby the glyph.
+SKILL_20160224_006235	SKILL_20160224_006235    Place a glyph on the ground symbolizing Ogoun Badagris, the loa of power. Increases STR and AoE attack ratio of your zombies nearby the glyph.
 SKILL_20160224_006236	STR: +#{CaptionRatio}#{nl}AoE Attack Ratio: +#{CaptionRatio2}#{nl}Duration: #{CaptionTime}# seconds{nl}Glyph Duration: 15 seconds
-SKILL_20160224_006237	Carve a statue of Zemyna, the goddess of earth. Decreases the SP cost and SP recovery time when using skills for nearby allies who are within the radius of the Goddess Statue.
+SKILL_20160224_006237	Carve a statue of Zemyna, goddess of the earth. Decreases the SP cost and SP recovery time when using skills for nearby allies who are within the radius of the Goddess Statue.
 SKILL_20160224_006238	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Additional Damage to Plant-type Monsters: +#{CaptionRatio}#{nl}Statue Materials Drop Rate: #{CaptionTime}#%
-SKILL_20160224_006239	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Strike down the enemy with a powerful attack. Deals double damage to Mutant and Devil-type enemies.
+SKILL_20160224_006239	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Strike down enemies with a powerful attack. Deals double damage to Mutant and Devil-type enemies.
 SKILL_20160224_006240	Attack: #{SkillAtkAdd}#{nl}Maximum Targets: #{CaptionRatio}#
 SKILL_20160224_006241	Attack per Hit: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Consumes 30 SP per second
-SKILL_20160224_006242	Reduced SP usage, deals additional damage when attacking.
+SKILL_20160224_006242	Reduced SP usage, and deals additional damage when attacking.
 SKILL_20160224_006243	Unable to move.
 SKILL_20160224_006244	High Anchoring: Critical Chance
 SKILL_20160224_006245	Rapid Fire: Critical Chance
-SKILL_20160224_006246	Increased maximum CON, defense, movement speed.
+SKILL_20160224_006246	Increased maximum HP, defense, and movement speed.
 SKILL_20160224_006247	Weak against physical property attacks.
 SKILL_20160224_006248	Langort: Blocking
-SKILL_20160224_006249	Swash Buckling: Buff
+SKILL_20160224_006249	Swash Buckling: Party Buff
```

Comment:
A few lines below is the actual tooltip: 

SKILL_20160224_006285	Party members will receive a buff when [Swash Buckling] is used.